src/test/java/nl/ordina/jobcrawler/service/SkillServiceTest.java

```diff
@@ -0,0 +1,297 @@
+package nl.ordina.jobcrawler.service;
+
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.repo.SkillRepository;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+
+class SkillServiceTest {
+
+    static private SkillRepository skillRepositoryMock;
+    static private SkillService skillService;
+
+    private Vacancy vacancy;
+    private List<Vacancy> vacanciesDB;  // fake "database" for vacancy
+    private String skillName;
+    private Skill skill;
+    private List<Skill> skillDB;    // fake "database" for skill
+
+    @BeforeAll
+    static void beforeAll() {
+        skillRepositoryMock = Mockito.mock(SkillRepository.class);
+        skillService = new SkillService(skillRepositoryMock);
```

Comment:
You might want to add these two to the method that is annotated with `@BeforeEach` instead of putting them in a `@BeforeAll` method.
When running all these test at once, `removeSkillsFromVacancy()` will fail. Running this test isolated, it will pass.

It looks to me that `@BeforeAll` will set these values only once. `@BeforeEach` will (re)set values for each test.